CHANGELOG.md

```diff
@@ -19,6 +20,11 @@ All notable changes to this project will be documented in this file. The format
 
 ## [Unreleased]
 
+## [1.3.12] - 2024-12-04
```

Comment:
Possibly needs to be changed, depending on when https://github.com/bitcoin-sv/arc/pull/611 will be merged